python/nano/src/bigdl/nano/pytorch/k8s/bigdl-submit.py

```diff
@@ -0,0 +1,109 @@
+
+import subprocess
+import sys
+import os
+from uuid import uuid4
+from kubernetes import client, config
+from argparse import REMAINDER, ArgumentParser
+
+def get_args_parser() -> ArgumentParser:
+    """Helper function parsing the command line options."""
+
+    parser = ArgumentParser(description="Nano Training Launcher")
+
+    parser.add_argument(
+        "--nnodes",
+        type=int,
+        default=1,
+        help="Number of nodes",
+    )
+
+    parser.add_argument(
+        "--image",
+        type=str,
+        default="10.239.45.10/arda/nano-run:latest",
+        help="Rank of the node for multi-node distributed training.",
+    )
+
+    parser.add_argument(
+        "--master_port",
+        type=str,
+        default="12345",
```

Comment:
Any recommended port range instead of 12345?

Reply:
I think maybe we can random select a port

Reply:
> I think maybe we can random select a port

random range maybe a good idea （12XXX）